app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def increment_post_comments_counter
+    post.comments_counter += 1
+    post.save
+  end
```

Comment:
- [ ] The same method you can use here as well. 
Just like we have in model/post.rb

`  after_save :update_post_counter
...
`

Reply:
Smart ✅

Reply:
This change has been done well.